configs/stage/roles/notifications.json

```diff
@@ -10,6 +10,9 @@
         {
           "permission": "notifications:*:*"
         },
+        {
+          "permission": "events:*:*"
```

Comment:
same

Reply:
not sure. That was my question as well. Maybe @theute knows if this is needed or not?

Reply:
I don't think we need it

Reply:
@josejulio is right, format is application:<resource type>:<verb>
https://consoledot.pages.redhat.com/docs/dev/services/rbac.html